0003-lightning-as-alpha-ledger.md

```diff
@@ -0,0 +1,158 @@
+
+2) Bob _accepts_ the request and performs the following step:
+    * Create a hold incoice using `addholdinvoice` with the amount `X` and the hashed preimage `secret_hash` ([more details](https://github.com/lightningnetwork/lnd/blob/aa1cd04dbf07a9195d5ada752f383988d8d01fa7/cmd/lncli/invoicesrpc_active.go#L142)).
+    * Bob _subscribes_ to the invoice and waits for the payment either using
+      * `SubscribeSingleInvoice` - this is not available through the CLI but as RPC; or
+      * `LookupInvoice` - this is available trough the CLI but needs to be polled regularly.
+3) Bob response to Alice's request with the following information:
+
+| Name                    | JSON Encoding | Description                                                                                               |
+|-------------------------|---------------|-----------------------------------------------------------------------------------------------------------|
+| `alpha_redeem_identity` | `α::Identity` | The identity on alpha that **A**, this is the LN `invoice`  |
+| `beta_refund_identity`  | `β::Identity` | The identity on beta that **B** will be transferred to when the beta-HTLC is activated after `beta_expiry`      |
+
+4) Alice now starts the [exection phase](https://github.com/comit-network/RFCs/blob/master/RFC-003-SWAP-Basic.md#1-alice-deploys-%CE%B1-htlc) by paying the invoice using the LND command `sendpayment`.
+5) Bob gets notified about funding of alpha (i.e. the invoice has been paid but cannot be settled yet), and continues with [deploying beta-HTLC](https://github.com/comit-network/RFCs/blob/master/RFC-003-SWAP-Basic.md#2-bob-deploys-%CE%B2-htlc), i.e. he deploys a HTLC on Ethereum.
+6) As soon as beta has enough confirmations for Alice, she redeems the beta-HTLC using her secret.
+7) Bob gets notified about this, learns the secret and can now settle the LND invoice by invoking the LND command `settleInvoice` ([more details](https://github.com/lightningnetwork/lnd/blob/aa1cd04dbf07a9195d5ada752f383988d8d01fa7/cmd/lncli/invoicesrpc_active.go#L53)).
+
+The trade is complete.
+
+### Considerations and limitations
+
+#### Lightning Network: a new ledger
+Similiar to the ledger definitions for [Bitcoin](https://github.com/comit-network/RFCs/blob/master/RFC-004-Bitcoin.md) and [Ethereum](https://github.com/comit-network/RFCs/blob/master/RFC-006-Ethereum.md) we need to handle the Lightning Network differently.
+This is required because the comit-node and btsieve need to perform different actions accordingly.
+We are always talking about Ledgers and Assets, (e.g. _Bitcoin_ Asset on the _Bitcoin_ Ledger, _Ether_ Asset on the _Ethereum_ Ledger, _Erc20_ on the _Ethereum_ Ledger, ...), Hence,
+if we follow this approach, for supporting LN (through LND) we will need to introduce a new pair of Ledger and Asset:
+
+* Ledger: the **Lightning Network**. _Ledgers_ are used as _settlement layers_ for our HTLCs. In the case of LND, this layer is the Lightning Network.
```

Comment:
One could argue whether it should be `LightningNetwork` or `LND`. I opt for `LightningNetwork` at this stage.

Reply:
I also opt for `LightningNetwork`.